error-prone-contrib/src/main/java/tech/picnic/errorprone/refastertemplates/OptionalTemplates.java

```diff
@@ -17,8 +17,11 @@
 import java.util.function.Supplier;
 import java.util.stream.Stream;
 import javax.annotation.Nullable;
+import tech.picnic.errorprone.refaster.annotation.Description;
+import tech.picnic.errorprone.refaster.annotation.OnlineDocumentation;
 
 /** Refaster templates related to expressions dealing with {@link Optional}s. */
+@OnlineDocumentation
```

Comment:
Here and below: don't forget to revert the changes in this file, assuming we introduce the annotations separately.